Makefile

```diff
@@ -48,14 +48,16 @@ extract_translations: ## extract strings to be translated, outputting .mo files
 
 fake_translations: extract_translations dummy_translations compile_translations ## generate and compile dummy translation files
 
-pip-compile: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
-	pip install --user -q pip-tools
+upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
+upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
```

Comment:
Won't only one of these two identical `upgrade` Makefile targets be executed?

Reply:
This (as I recently discovered) is the syntax for a target-specific `make` variable: https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html .  We've started using this in a few repos to set this environment variable for all calls to `pip-compile` within the `upgrade` target, so the instructions at the top of each generated requirements file are correct.

Reply:
Ah, `make`. Providing obscure syntax and features since forever... 📆